frontend/app/src/premium/premium-apis.ts

```diff
@@ -40,22 +38,23 @@ import { useSessionSettingsStore } from '@/store/settings/session';
 import { useAdexStakingStore } from '@/store/staking';
 import { useStatisticsStore } from '@/store/statistics';
 import { One } from '@/utils/bignumbers';
+import { isNft } from '@/utils/nft';
 
 export const assetsApi = (): AssetsApi => {
-  const { assetInfo, assetSymbol, tokenAddress } = useAssetInfoRetrieval();
-
-  const { getNftDetails } = useNftAssetInfoStore();
+  const { assetInfo, assetSymbol, assetName, tokenAddress } =
+    useAssetInfoRetrieval();
 
   return {
     assetInfo: (identifier: MaybeRef<string>) =>
       computed(() => {
-        const nft = get(getNftDetails(identifier)) as AssetInfo | null;
-        return nft ?? get(assetInfo(identifier));
+        return get(assetInfo(identifier));
```

Comment:
Since we don't have any logic in here anymore can't we do either
`assetInfo: (identifier: MaybeRef<string>) => assetInfo(identifier)` or just `assetInfo,`. Wrapping in a computed doesn't seem needed anymore.